[IMP] empty

arpi-odoo opened the pull request.

arpi-odoo committed:
[FIX] hr: new contract button invisible

arpi-odoo committed:
[IMP] hr_skills: cleanup certification reporting

arpi-odoo committed:
[IMP] hr_payroll: constraint positive wage

arpi-odoo committed:
[FIX] hr_holidays: fix traceback leave type

arpi-odoo committed:
[FIX] hr: fix delete version

arpi-odoo committed:
[FIX] hr: fix various bugs

HashemKhaled committed:
[FIX] hr: hide new contract button if there is no contract dates

This hides the New Contract button when there are no contracts associated with the employee.

Task: 5231904

arpi-odoo committed:
[IMP] empty

robodoo commented:
This PR targets the un-managed branch odoo-dev/odoo:master-hr-onboarding, it needs to be retargeted before it can be merged.

HashemKhaled committed:
[FIX] hr: allow version deletion for employees with multiple versions

This makes it possible to delete a version if there are multiple versions for one employee.

Task: 5232093

HashemKhaled committed:
[FIX] hr_holidays: fix search on leave type

**Steps to reproduce:**
Navigate to Time Off >> Configuration >> Time Off Types >> Open any type, navigate to the Smart button Time Off >> Click on New >> Time Off Type
Leads to an error.

Reference Video: https://drive.google.com/file/d/1vi_apL24Km5tSomQtwcTArqqWvrBR5ni/view?usp=drive_link

**Reason:**
The `op` function is a comparison operator function that expects two values to compare and we are only passing to it `leave_type.virtual_remaining_leaves` parameter.

**Solution:**
Added the missing `value` parameter.

Task: 5238200

HashemKhaled committed:
[CLN] hr_skills: cleanup cerification reporting

Due to LMS (Learning Management System) and Skills refactor we ended up having two reports about certifications. This PR cleans it.

Reference: https://tinyurl.com/2cf2ogoq

Task: 5246246
